### ISSUE TITLE
[Feature request] web gui to enable hosting on a dedicated server 

### ISSUE THREAD

**Issue by chappyj** [shdwp/dcs_liberation#107]
Wondering if it would be worth looking at a web based gui that allows the same single player functionality shared by users online that would generate the miz for a dedicated server. Users could participate in the operational decisions and flight planning

**Comment by wrycu** [comment]
I agree with the concept, but it would be a lot of work and I'm not sure much value gain there is over simply streaming your screen with Liberation open.
